doc/en/reference/customize.rst

```diff
@@ -20,8 +20,7 @@ Configuration file formats
 --------------------------
 
 Many :ref:`pytest settings <ini options ref>` can be set in a *configuration file*, which
-by convention resides on the root of your repository or in your
-tests folder.
+by convention resides on the root of your repository.
```

Comment:
I think "in the root directory of your repository" would be more idiomatic

Reply:
English isn't my native language (and I'm not the best student in the English class), so not objections.